src/RA_Core.cpp

```diff
@@ -852,20 +855,25 @@ API void CCONV _RA_CheckForUpdate()
         }
         else
         {
+            // basic_ostringstream would seem overkill here
+            tstring tstr{ TEXT("Error in download from ") };
+            tstr.append(RA_HOST_URL);
+            tstr.append(TEXT("...\n"));
+            tstr.append(TEXT("Please check your connection settings or RA forums!\n"));
```

Comment:
Similar comment about string building. Will this get resolved at compile time?

Reply:
std::string cannot be built in compile time, it's just not possible. const char*, const wchar_t can be but it has to be in the same expression (constexpr).